wdl/transforms/new-base/src/test/scala/wdl/transforms/base/linking/expression/values/EngineFunctionEvaluatorsSpec.scala

```diff
@@ -0,0 +1,29 @@
+package wdl.transforms.base.linking.expression.values
+
+import org.scalatest.flatspec.AnyFlatSpec
+import org.scalatest.matchers.should.Matchers
+
+class EngineFunctionEvaluatorsSpec extends AnyFlatSpec with Matchers{
+  behavior of "read_lines"
+  it should "return an empty array with an empty file" in {
```

Comment:
I would suggest replacing `with` with `from` throughout this test set

```suggestion
  it should "return an empty array from an empty file" in {
```